panther_bringup/README.md

```diff
@@ -1,162 +1,9 @@
-- `/tf` [*tf2_msgs/TFMessage*]: publishes `odom` to `base_link` transform.
-
-[//]: # (ROS_API_NODE_PUBLISHERS_END)
-
-#### Service Servers
-
-[//]: # (ROS_API_NODE_SERVICE_SERVERS_START)
-
-- `localization/set_pose` [*robot_localization/SetPose*]: by issuing a *geometry_msgs/PoseWithCovarianceStamped* message to the set_pose topic, users can manually set the state of the filter. This is useful for resetting the filter during testing and allows for interaction with RViz. Alternatively, the state estimation nodes advertise a SetPose service, whose type is *robot_localization/SetPose*.
-
-[//]: # (ROS_API_NODE_SERVICE_SERVERS_END)
-[//]: # (ROS_API_NODE_END)
-
-[//]: # (ROS_API_NODE_START)
-
-[//]: # (ROS_API_NODE_COMPATIBLE_1_0)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-
-[//]: # (ROS_API_NODE_NAME_START)
-
-### imu_filter_node
-
-[//]: # (ROS_API_NODE_NAME_END)
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
-
-External node type: *[imu_filter_madgwick/imu_filter_node](https://github.com/CCNYRoboticsLab/imu_tools/blob/humble/imu_filter_madgwick/src/imu_filter_node.cpp)*.
-
-Node responsible for filtering and fusing raw data from the IMU.
-
-[//]: # (ROS_API_NODE_DESCRIPTION_END)
-
-#### Subscribers
-
-[//]: # (ROS_API_NODE_SUBSCRIBERS_START)
-
-- `~/imu/data_raw` [*sensor_msgs/Imu*]: the raw accelerometer and gyroscope data.
-
-[//]: # (ROS_API_NODE_SUBSCRIBERS_END)
-
-#### Publishers
-
-[//]: # (ROS_API_NODE_PUBLISHERS_START)
-
-- `~/imu/data` [*sensor_msgs/Imu*]: the fused IMU messages, containing the orientation.
-
-[//]: # (ROS_API_NODE_PUBLISHERS_END)
-[//]: # (ROS_API_NODE_END)
-
-[//]: # (ROS_API_NODE_START)
-
-[//]: # (ROS_API_NODE_COMPATIBLE_1_0)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-
-[//]: # (ROS_API_NODE_NAME_START)
-
-### phidgets_spatial_node
-
-[//]: # (ROS_API_NODE_NAME_END)
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
-
-External node type: *[phidgets_spatial/spatial_ros_i.cpp](https://github.com/ros-drivers/phidgets_drivers/blob/humble/phidgets_spatial/src/spatial_ros_i.cpp)*.
-
-The ROS driver for Phidgets Spatial.
-
-[//]: # (ROS_API_NODE_DESCRIPTION_END)
-
-#### Publishers
-
-[//]: # (ROS_API_NODE_PUBLISHERS_START)
-
-- `~/imu/data_raw` [*sensor_msgs/Imu*]: the raw accelerometer and gyroscope data.
-- `~/imu/is_calibrated` [*std_msgs/Bool*]: whether the gyroscope has been calibrated. This will be done automatically at startup time but can also be re-done at any time by calling the `imu/calibrate` service.
-- `~/imu/mag` [*sensor_msgs/MagneticField*]: the raw magnetometer data.
-
-[//]: # (ROS_API_NODE_PUBLISHERS_END)
-
-#### Service Servers
-
-[//]: # (ROS_API_NODE_SERVICE_SERVERS_START)
-
-- `~/imu/calibrate` [*std_srvs/Empty*]: run calibration on the gyroscope.
+## Launch Files
 
-[//]: # (ROS_API_NODE_SERVICE_SERVERS_END)
-[//]: # (ROS_API_NODE_END)
+This package contains:
 
-[//]: # (ROS_API_PACKAGE_END)
+- `bringup.launch.py`: Responsible for activating whole robot system.
```

Comment:
I'm not sure why we moved the whole bringup package description to master readme file?

Reply:
I don't know what things you're talking about, but keeping additional issues in bringup in the previous version would cause a lot of redundancy. I will divide the answer into 3 points because previously bringup contained 3 things:

1. Default Nodes Launched

If we wanted to maintain the standard in all packs, the bringup and simulation launch packages would be huge and would contain descriptions of nodes from other packages because they are used.

2. Bringup Launch Arguments

Similarly to the above, bringup and simulation have almost identical arguments and it is best to indicate this in the place where they are launched. There would also be a lot of redundancy because we would have to describe namespace and use_sim args in almost every file.

3. Description of nodes
Currently, bringup does not fire any nodes, so it is not needed.